tests/fixtures/pipeline/insufficient_metrics.json

```diff
@@ -0,0 +1,311 @@
+{
+  "resource_usage": [
+    {
+      "executors": [
+        {
+          "allocated": [
+            {
+              "name": "cpus",
+              "revocable": {},
+              "role": "*",
+              "scalar": {
+                "value": 1
+              },
+              "type": "SCALAR"
+            },
+            {
+              "name": "mem",
+              "role": "*",
+              "scalar": {
+                "value": 160
+              },
+              "type": "SCALAR"
+            }
+          ],
+          "executor_info": {
+            "command": {
+              "shell": true,
+              "value": "/home/mesos/usr/libexec/mesos/mesos-executor"
+            },
+            "executor_id": {
+              "value": "serenityBe2"
+            },
+            "framework_id": {
+              "value": "20150622-161150-16842879-5050-16372-0000"
+            },
+            "name": "Command Executor (Task: serenityBe2) (Command: sh -c 'watch pwd')",
```

Comment:
I ment to check if our modules will not fail for not proper ResourceUsage. Here, there are no total resources.